src/index.html

```diff
@@ -3,82 +3,90 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="A project that hightlights a11y fails, created by @5t3ph">
     <link
       href="https://fonts.googleapis.com/css2?family=Rowdies:wght@700&display=swap"
       rel="stylesheet"
     />
     <link rel="stylesheet" href="/css/style.css" />
+    <title>A11y Fails</title>
   </head>
   <body>
-    <ul class="nav">
-      <li><a href="/about">About</a></li>
-      <li><a href="/contact">Contact</a></li>
-      <li><a href="/products">Products</a></li>
-    </ul>
-    <div class="header">
+    <a href="#main" class="skip-link">Skip to main content</a>
+    <nav role="navigation">
+      <ul class="nav">
+        <li><a href="/about">About</a></li>
+        <li><a href="/contact">Contact</a></li>
+        <li><a href="/products">Products</a></li>
+      </ul>
+    </nav>
+    <header class="header">
       <h1><span>a11y</span> Fails</h1>
-    </div>
-    <div class="main">
-      <h3>Product List</h3>
-      <aside class="products">
-        <div class="card">
+    </header>
+    <main class="main" role="main" id="main" tabindex="-1">
```

Comment:
Same idea here as `nav` where `role` is redundant 👍

Reply:
Thanks 👍 fixed with commit `82fa62b`